client/components/dot-pager/docs/example.jsx

```diff
@@ -2,7 +2,7 @@ import DotPager from '../';
 
 export default function DotPagerExample() {
 	return (
-		<DotPager>
+		<DotPager hasDynamicHeight={ true }>
```

Comment:
`={ true }` is redundant.

```suggestion
		<DotPager hasDynamicHeight>
```